share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -78,14 +78,39 @@ default:
       when: never
     - if: $SPACK_CI_ENABLE_STACKS =~ /.+/ && $SPACK_CI_STACK_NAME !~ $SPACK_CI_ENABLE_STACKS
       when: never
-    - if: $CI_COMMIT_REF_NAME == "develop" || $CI_COMMIT_REF_NAME =~ /^releases\/v.*/
-      # Pipelines on develop/release branches only rebuild what is missing from the mirror
```

Comment:
@scottwittenburg I think I maybe was missing something from develop. Did you do this?

Reply:
Yeah, release/v* are like develop